hydradx/notebooks/Omnipool/LP_fees_analysis.ipynb

```diff
@@ -0,0 +1,269 @@
+    "        ),\n",
+    "        'LP': Agent(\n",
+    "            holdings={'TKN': 10000},\n",
+    "            trade_strategy=invest_all('omnipool')\n",
+    "        ),\n",
+    "        # 'LP2': Agent(\n",
+    "        #     holdings={'DOT': 10000},\n",
+    "        #     trade_strategy=invest_all('omnipool')\n",
+    "        # )\n",
+    "    },\n",
+    "    external_market={tkn: assets[tkn]['usd price'] for tkn in assets},\n",
+    "    archive_all=False\n",
+    ")\n",
+    "\n",
+    "# copy DOT prices for TKN\n",
+    "# for j, step in enumerate(price_list):     \n",
+    "#     step['TKN'] = step['DOT'] / price_list[0]['DOT']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "b3f08712-922b-465f-b04b-5f3057a78ab5",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-05-24T15:14:34.207704241Z",
+     "start_time": "2023-05-24T15:01:10.779629649Z"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting simulation...\n",
+      "Execution time: 42.282 seconds.\n",
+      "Trade volume per day as a fraction of TVL: {'HDX': 0.0, 'USD': 0.0, 'BTC': 0.0, 'ETH': 0.0, 'DOT': 0.0, 'TKN': 0.0}\n",
+      "Starting simulation...\n",
+      "Execution time: 45.922 seconds.\n",
+      "Trade volume per day as a fraction of TVL: {'HDX': 7.721712058478256e-05, 'USD': 0.0007576275147206286, 'BTC': 25.148787191903676, 'ETH': 0.4596912646564797, 'DOT': 0.0009166227365982178, 'TKN': 0.004978545123360402}\n",
+      "Starting simulation...\n",
+      "Execution time: 45.678 seconds.\n",
+      "Trade volume per day as a fraction of TVL: {'HDX': 0.00015574336368770852, 'USD': 0.0015028036009165375, 'BTC': 51.4861193881495, 'ETH': 0.9115494465340751, 'DOT': 0.0018153983689646843, 'TKN': 0.009859476554592293}\n",
+      "Starting simulation...\n",
+      "Execution time: 43.363 seconds.\n",
+      "Trade volume per day as a fraction of TVL: {'HDX': 0.0002324028364840227, 'USD': 0.0022956224009681946, 'BTC': 76.80552149768843, 'ETH': 1.3874016659422124, 'DOT': 0.0027226256534754016, 'TKN': 0.015107392684884846}\n",
+      "Starting simulation...\n",
+      "Execution time: 46.242 seconds.\n",
+      "Trade volume per day as a fraction of TVL: {'HDX': 0.0003084036793473525, 'USD': 0.003019164687591154, 'BTC': 100.60900592558187, 'ETH': 1.8271961702455901, 'DOT': 0.0035806015643876144, 'TKN': 0.02007961106980003}\n"
+     ]
+    }
+   ],
+   "source": [
+    "time_steps = 219000  # int(7200 * 365 / 12)\n",
+    "volume_events = []\n",
+    "trade_volume = []\n",
+    "semi_final_state = []\n",
+    "final_state = []\n",
+    "# initial_state._evolve_function = historical_prices(price_list)\n",
+    "for i in range(5):\n",
+    "    initial_state.agents['Trader'].trade_strategy=price_sensitive_trading(\n",
+    "        pool_id='omnipool',\n",
+    "        max_volume_usd=1.65 * i,\n",
```

Comment:
It's an arbitrary number that I determined would give the right trade volume. I didn't like that, because it's imprecise, so I changed the trade strategy to back-and-forth, which can be more accurate.